src/services/version-control-system-integration-services/list-repositories-service.js

```diff
@@ -35,7 +37,10 @@ const parseHttpResponse = (httpResponse) => {
       return httpResponse.body
     case 400:
       let errorMessage = httpResponse.body.error
-      errorMessage = errorMessage === 'Invalid scope.' ? 'Please, try another Scope.' : errorMessage
+      errorMessage =
+        errorMessage === 'Invalid scope.'
+          ? 'Your scope does not contain any valid repositories.'
+          : errorMessage
       throw new Error(errorMessage).message
```

Comment:
Sem as chaves, faz o linter acusar "Unexpected lexical declaration in case block" devido à declaração de variáveis dentro do case

```suggestion
 case 400: {
      let errorMessage = httpResponse.body.error
      errorMessage =
        errorMessage === 'Invalid scope.'
          ? 'Your scope does not contain any valid repositories.'
          : errorMessage
      throw new Error(errorMessage).message
}
```

Reply:
Adicionado, mas não tenho tanta certeza sobre o linter, já que praticamente todos os nossos services possuem o parseHttpResponse dessa maneira.